Documentation/environment-variables.rst

```diff
@@ -7,28 +7,27 @@ Communication Thread
 --------------------
 
 By default, any SeisSol run with more than one MPI rank will use a communication thread to advance the MPI progress engine.
-For that, you will need to leave at least one thread vacant in your OpenMP thread placing map, cf. the SuperMUC-NG example below.
+For that, you must leave at least one thread vacant in your OpenMP thread placing map, cf. the SuperMUC-NG example below.
 
-If you do not want to use a communication thread, you may set `SEISSOL_COMMTHREAD=0`; then SeisSol polls on the progress from time to time.
+If you do not want to use a communication thread, you may set `SEISSOL_COMMTHREAD=0`; then, SeisSol polls the progress from time to time.
```

Comment:
You can use ` `` ` here for the `SEISSOL_COMMTHREAD=0` . Also, how about "you may set `SEISSOL_COMMTHREAD=0` to make SeisSol poll the progress [...]" ? (just to remove the semicolon after the code snippet)

Reply:
What do you mean? `` is already used? Or did you mean to use ``SEISSOL_COMMTHREAD=0``?

Reply:
The `rst` syntax will interpret a single
```
`
```
as italic text; only a double ` `` ` as code

Reply:
Understood. Thanks! Modified